spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/datadog/DatadogProperties.java

```diff
@@ -33,6 +33,18 @@
 	 */
 	private String apiKey;
 
+	/**
+	 * Datadog application key. Not strictly required, but improves the Datadog
+	 * experience by sending meter descriptions, types, and base units to Datadog.
+	 */
+	private String applicationKey;
+
+	/**
+	 * Enable publishing descriptions metadata to Datadog. Turn
+	 * this off to minimize the amount of metadata sent.
+	 */
+	private Boolean descriptions;
```

Comment:
Nothing is using that property as far as I know. Am I missing something?

Reply:
`DatadogMetricMetadata` is.

Reply:
Fixed with 447ddea92e